src/modules/MouseUtils/MouseHighlighter/MouseHighlighter.cpp

```diff
@@ -330,6 +342,23 @@ LRESULT CALLBACK Highlighter::WndProc(HWND hWnd, UINT message, WPARAM wParam, LP
     case WM_DESTROY:
         instance->DestroyHighlighter();
         break;
+    case WM_TIMER:
+    {
+        switch (wParam)
+        {
+        case BRING_TO_FRONT_TIMER_ID:
+            static int fireCount = 0;
+            if (fireCount++ >= 4)
```

Comment:
Why are we firing the timer 4 times before de-registering?
Also the description is outdated regarding what the PR is doing, right?

Reply:
I've updated the description, thanks for the hint.
My experience is that it takes 0-30ms that our window gets hidden by the pinned window. If we use a simple timer with one period of 50ms, there is a little vibration on the UI, as in most of the cases our circle gets hidden in a few miliseconds. One timer with a short period is not good enough, in 5% of the cases the pinned window will hide our window. To have a flickeringless UI, we bring our window to top position 5 times (with 10 ms delay between them).

Reply:
Let's add some comments to explain it, then, please 😉